client/src/pages/Submit/Submit.tsx

```diff
@@ -0,0 +1,75 @@
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import React, { useEffect, useState } from 'react';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Grid from '@material-ui/core/Grid';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { Box } from '@material-ui/core';
+import Navbar from '../../components/Navbar/Navbar';
+import { uploadImage } from '../../helpers/APICalls/uploadImage';
+import { useHistory } from 'react-router-dom';
+import useStyles from './useStyles';
+export default function Submit(): JSX.Element {
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+  const classes = useStyles();
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    return <CircularProgress />;
+  }
+  return (
+    <Grid container>
+      <CssBaseline />
+      <Box className={classes.contentWrapper}>
```

Comment:
MUI Box has flexbox built into it, just add a prop to this component rather than a class 

Eg: `<Box display="flex"></Box>`